onnxruntime_extensions/tools/add_pre_post_processing_to_model.py

```diff
@@ -20,6 +20,48 @@ class ModelSource(enum.Enum):
     OTHER = 2
 
 
+def image_processor(extend_config: dict):
```

Comment:
I think we're getting to a point where the complexity of this script in trying to handle so many different models is making it far less useful as a reference to someone who wants to create a custom pre/post processing pipeline or tweak an existing one. I'd like to start splitting it up into per-model examples (e.g. add_pre_post_processing_to_mobilenet) to simplify. 

Can we take that approach for CLIPImageProcessor and have it in a completely separate script?

Slightly different as it's a more generic processor and not model specific, so maybe called something like add_HuggingFace_CLIPImageProcessor_to_model.py?

Reply:
"Transformers" and "Clip processor" was spinning  off from `add_pre_post_processing_to_model.py`. The usage is not changed.

Reply:
Sorry - I don't think I was clear enough. I don't want the usage of add_pre_post_processing_to_model.py to be unchanged. I want to start breaking that up into simpler scripts so the pre/post processing pipeline is defined as directly as possible. It's much harder for a new user to see what is happening when they have to understand an extra layer of configuration. Main goal is for these to be reference implementations for external users.

With more model specific scripts I would hope it looks more like this example for a super-resolution model with different pre/post processing to the super resolution model we handle here currently:

https://github.com/microsoft/onnxruntime-inference-examples/blob/39d587857554451976fba379245a78184ac7d82d/mobile/examples/super_resolution/MAUI/real_esrgan_ort_e2e.py#L68-L92

That would mean that we leave what is in the existing script alone for now, and create a new script to handle just ClipImageProcessor. 

Does that make sense?

Reply:
Make sence.